src/safeposix/syscalls/net_calls.rs

```diff
@@ -1563,18 +1629,27 @@ impl Cage {
                                 }
                             }
 
-                            let ladr = sockhandle.localaddr.unwrap().clone(); //must have been populated by implicit bind
+                            //The socket must have been assigned an address by implicit bind
+                            let ladr = sockhandle.localaddr.unwrap().clone();
+                            //Grab a tuple of the address, port, and port type
+                            //to be inserted into the set of listening ports
                             let porttuple = mux_port(
                                 ladr.addr().clone(),
                                 ladr.port(),
                                 sockhandle.domain,
                                 TCPPORT,
                             );
 
+                            //Set the socket connection state to listening
+                            //to readily accept connections
                             NET_METADATA.listening_port_set.insert(porttuple.clone());
                             sockhandle.state = ConnState::LISTEN;
 
-                            let listenret = sockhandle.innersocket.as_ref().unwrap().listen(5); //default backlog in repy for whatever reason, we replicate it
+                            //Call listen from libc on the socket
+                            //Set the backlog to 5:
+                            //default backlog in repy for whatever reason, we replicate it
+                            //** Would we ever want to change the backlog??  **/
+                            let listenret = sockhandle.innersocket.as_ref().unwrap().listen(5);
```

Comment:
But we do have a backlog argument passed into listen_syscall, why not just use it directly?

Reply:
Good question. @rennergade why was the backlog set to 5 and not the number passed in to listen?

Reply:
bumping this up @rennergade

Reply:
Yeah i dont see any reason to not use the passed parameter. Lets make that change. Just a weird legacy thing.